features/step_definitions/journey_test_steps.rb

```diff
@@ -1471,15 +1472,27 @@ def filter_to_check(filter_name,filter_value,filter_exist)
   page.should have_selector(:xpath, ".//*[@id='global-breadcrumb']/nav/*[@role='breadcrumbs']/li[1]//*[contains(text(), 'Admin home')]")
 end
 
-Then /I am presented with the G-Cloud 7 Statistics page$/ do
-  page.find(
-    :xpath,
-    "//p[contains(text(), 'G-Cloud 7')]/../h1[contains(text(), 'Statistics')]"
-    )
-    current_url.should end_with("#{dm_frontend_domain}/admin/statistics/g-cloud-7")
+Then /I am presented with the '(.*)' page$/ do |statistics_page|
```

Comment:
As this step definition is only for statistics pages it might be better to move the 'statistics' part out of the variable and just have the framework name captured inside the quotes - e.g. 
`/I am presented with the '(.*)' statistics page$/ do |framework_name|`

Because as it is it looks like this step could be used for any type of page in the scenarios, but e.g. it looks like you could do something like `Then I am presented with the 'G-Cloud 7 dashboard' page` or whatever.